### PR TITLE
Update README and interactive Docker serve

### LINKED ISSUE #27: README.md needs updating
Currently it's the same as that for deis/workflow.

### PR DESCRIPTION
**Pull request by slack**
We're just docs now!

Add `Makefile` targets for docker-based and local documentation options.

Fixes #27

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,66 +1,35 @@
...
 
-[![Build Status](https://travis-ci.org/deis/workflow.svg?branch=master)](https://travis-ci.org/deis/workflow) [![Go Report Card](http://goreportcard.com/badge/deis/workflow)](http://goreportcard.com/report/deis/workflow)
+## About
 
 Deis (pronounced DAY-iss) is an open source PaaS that makes it easy to deploy and manage
 applications on your own servers. Deis builds on [Kubernetes](http://kubernetes.io/) to provide
 a lightweight, [Heroku-inspired](http://heroku.com) workflow.
 
-## Work in Progress
+This repository represents the documentation for Deis Workflow which is the second major relase of the Platform.
```
Comment (carmstrong) [inline README.md]: s/relase/release/

File: README.md [hunk 2]
```diff
@@ -1,66 +1,36 @@
...
-## Work in Progress
+This repository represents the documentation for Deis Workflow which is the
+second major release of the Platform.
 
-![Deis Graphic](https://s3-us-west-2.amazonaws.com/get-deis/deis-graphic-small.png)
+## Requirements
 
-Deis Workflow v2 is currently in alpha. Your feedback and participation are more than welcome, but be
-aware that this project is considered a work in progress.
+The documentation site requires either a local installation of [mkdocs][] or
```
Comment (mboersma) [inline README.md]: This link to MkDocs isn't connected to a URL, but should point to http://www.mkdocs.org/. Also they seem to prefer `MkDocs` with the StudlyCaps / CamelCase.

**Comment by mboersma** [comment]
I tested the new Makefile targets and proofread. Everything looks great but for the MkDocs link I mentioned above.

File: README.md [hunk 3]
```diff
@@ -1,66 +1,36 @@
...
+## Serve Documentation
 
-```console
-$ make docker-build docker-push
-```
+To serve documenation run: `make serve` or `make docker-serve`
 
-You'll want to modify the deis chart to use your custom image, then run `helm install` on the
-chart.
+Then view the documentation on [http://localhost:8000](http://localhost:8000) or [http://DOCKER_IP:8000](http://DOCKER_IP:8000)
```
Comment (bacongobbler) [inline README.md]: I think we should just backquote `http://DOCKER_IP:8000` rather than a broken link.

**Pull request merged**